src/main/java/racingcar/RacingMain.java

```diff
@@ -0,0 +1,33 @@
+package racingcar;
+
+import racingcar.domain.RacingcarGame;
+import racingcar.view.InputView;
+import racingcar.view.OutputView;
+
+public class RacingMain {
+
+    private static final InputView inputView = new InputView();
+    private static final OutputView outputView = new OutputView();
+
+    public static void main(String[] args) {
+        outputView.printGuide();
+        String names = inputView.getNames();
+
+        outputView.printTryCount();
+        int tryCount = inputView.getNumber();
+
+        playGame(tryCount, names);
+    }
+
+    private static void playGame(int tryCount, String names) {
+        outputView.printResultGuide();
+
+        RacingcarGame racingcarGame = new RacingcarGame(names, tryCount);
+        while (!racingcarGame.isEnd()) {
```

Comment:
사소하지만 이런 케이스는 긍정문이 조금 더 잘 읽히기 때문에 크게 공수가드는 것이 아니라면 긍정문을 사용하시는걸 추천드려요. 😉

부정ex) 레이싱 게임이 아니 끝났다. 
긍정ex) 레이싱 게임이 진행중이다.

```suggestion
        while (racingcarGame.isPlaying()) {
```